[2023-10-01T02:29:49.621+0300] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dummy_train_dag.data_preprocessing scheduled__2023-09-29T01:00:00+00:00 [queued]>
[2023-10-01T02:29:49.625+0300] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dummy_train_dag.data_preprocessing scheduled__2023-09-29T01:00:00+00:00 [queued]>
[2023-10-01T02:29:49.625+0300] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-01T02:29:49.706+0300] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): data_preprocessing> on 2023-09-29 01:00:00+00:00
[2023-10-01T02:29:49.714+0300] {standard_task_runner.py:57} INFO - Started process 23698 to run task
[2023-10-01T02:29:49.717+0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'dummy_train_dag', 'data_preprocessing', 'scheduled__2023-09-29T01:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_for_model_train.py', '--cfg-path', '/tmp/tmp3ohsodbb']
[2023-10-01T02:29:49.719+0300] {standard_task_runner.py:85} INFO - Job 43: Subtask data_preprocessing
[2023-10-01T02:29:49.766+0300] {task_command.py:415} INFO - Running <TaskInstance: dummy_train_dag.data_preprocessing scheduled__2023-09-29T01:00:00+00:00 [running]> on host dsbulgako1-QEMU-Virtual-Machine
[2023-10-01T02:29:49.844+0300] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='dmitrii.bulghakov@gmail.com' AIRFLOW_CTX_DAG_OWNER='Dmitry Bulgakov' AIRFLOW_CTX_DAG_ID='dummy_train_dag' AIRFLOW_CTX_TASK_ID='data_preprocessing' AIRFLOW_CTX_EXECUTION_DATE='2023-09-29T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-29T01:00:00+00:00'
[2023-10-01T02:29:49.845+0300] {s3.py:1359} INFO - This function shadows the 'download_file' method of S3 API, but it is not the same. If you want to use the original method from S3 API, please call 'S3Hook.get_conn().download_file()'
[2023-10-01T02:29:49.845+0300] {s3.py:1365} INFO - Downloading source S3 file from Bucket mlops-project with path mlops_project/california_housing.pkl
[2023-10-01T02:29:49.857+0300] {base.py:73} INFO - Using connection ID 's3_connector' for task execution.
[2023-10-01T02:29:49.858+0300] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='s3_connector', conn_type='aws') credentials retrieved from login and password.
[2023-10-01T02:29:49.899+0300] {configprovider.py:949} INFO - Found endpoint for s3 via: config_global.
[2023-10-01T02:29:50.567+0300] {configprovider.py:949} INFO - Found endpoint for s3 via: config_global.
[2023-10-01T02:29:51.192+0300] {test_dag_for_model_train.py:97} INFO - Data preparation finished!
[2023-10-01T02:29:51.193+0300] {python.py:194} INFO - Done. Returned value was: None
[2023-10-01T02:29:51.206+0300] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=dummy_train_dag, task_id=data_preprocessing, execution_date=20230929T010000, start_date=20230930T232949, end_date=20230930T232951
[2023-10-01T02:29:51.220+0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-01T02:29:51.239+0300] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
